### PR TITLE
Bootstrap upgrade to V4.3.1

### PR DESCRIPTION
**Pull request by ShmoogleOsukami**
This PR updates bootstrap to version 4.3.1 and switched to using a CDN source.
EDIT: It also enables livereload, It does give an depreciation error which needs sorting eventually.

It only really broke the tables which have been fixed.

The new bootstrap has a lot more to offer and better responsive classes, as such I'll eventually go though page by page getting everything updated and cleaned up.

As for the CDN sources, It allows for better cacheability and in turn can reduce the number of requests required to load the page.

### CODE REVIEW

**Comment by ShmoogleOsukami** [comment]
I have a feeling livereload was designed to make development easier and is not actually required in deployment. 
Let me know if that's the case and I'll get it took out, it's only 1 line in config.rb

File: source/Guides/whitelist.htm.erb [hunk 1]
```diff
@@ -58,10 +58,7 @@ description: "The Gridcoin whitelist is a list of community-approved BOINC proje
                                 Sponsor
                             </th>
                             <th>
-                                CPU
-                            </th>
-                            <th>
-                                GPU
+                                Compatibility
```
Comment (barton2526) [inline source/Guides/whitelist.htm.erb]: Why this change?

File: source/Guides/whitelist.htm.erb [hunk 2]
```diff
@@ -109,8 +106,8 @@ description: "The Gridcoin whitelist is a list of community-approved BOINC proje
                 </table>
             </div>
 
-            <div class="hidden-xs-down col-sm-12 hidden-md-up">
-                <table class="table table-hover table-condensed table-responsive">
+            <div class="col-12 d-none d-sm-block d-md-none">
+                <table class="table table-sm table-dark table-hover table-striped table-boreded">
```
Comment (barton2526) [inline source/Guides/whitelist.htm.erb]: Typo?  should be `table-bordered`

File: source/index.htm.erb [hunk 3]
```diff
@@ -82,7 +108,7 @@ description: "Gridcoin is a cryptocurrency which rewards volunteer distributed c
                         </tr>
                         <tr>
                             <td>
-                                Earn 10 GRC per block staked
+                                Constant Block Rewards
```
Comment (barton2526) [inline source/index.htm.erb]: Might need to be phrased differently.  I think the old version leads to less confusion, but am open to a different format.

File: source/layouts/layout.erb [hunk 4]
```diff
@@ -7,11 +7,17 @@
 	<%= partial "/layouts/partials/footer" %>
 
 	<!--   Core JS Files   -->
-	<script src="/assets/js/core/jquery.3.2.1.min.js" type="text/javascript"></script>
-	<script src="/assets/js/core/tether.min.js" type="text/javascript"></script>
-	<script src="/assets/js/core/bootstrap.min.js" type="text/javascript"></script>
+	<script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
```
Comment (barton2526) [inline source/layouts/layout.erb]: I'm not sure our webserver will be okay with cross origin requests.  We may need to verify this ourselves.

File: source/layouts/layout.erb [hunk 5]
```diff
@@ -7,11 +7,17 @@
...
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
+	<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
```
Comment (barton2526) [inline source/layouts/layout.erb]: why are we mixing sha256 and sha384?

**Comment by barton2526** [comment]
@ShmoogleOsukami I don't know if I like the idea of using cross origin references for the js files.  They are minified already and quite small.  Is the CDN going to really make enough difference to outweigh the potential issues associated with cross origin refs?

**Comment by barton2526** [comment]
Also I don't think `crossorigin="anonymous"` is supported in IE:
https://developer.mozilla.org/en-US/docs/Web/HTML/CORS_settings_attributes

**Comment by barton2526** [comment]
This should also be followed: https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity

File: source/Guides/whitelist.htm.erb (hunk 1 above)
Comment (ShmoogleOsukami) [reply source/Guides/whitelist.htm.erb]: Ahh that was residual changes from my whitelist update branch, missed that bit

File: source/layouts/layout.erb (hunk 5 above)
Comment (ShmoogleOsukami) [reply source/layouts/layout.erb]: I don't know, They are copy pasted from the bootstrap website, I'm guessing jquery only use sha256 where cloudflare and stackpath in force sha384

File: source/index.htm.erb (hunk 3 above)
Comment (ShmoogleOsukami) [reply source/index.htm.erb]: I could add a little hover notation to it so users can hover over for a brief explanation. To help reduce confusion.

File: source/layouts/layout.erb (hunk 4 above)
Comment (ShmoogleOsukami) [reply source/layouts/layout.erb]: With regards to the CDN stuff, I don't think there will be a huge performance gain at all really, Since the site is really lightweight and static. 
For the sake of simplicity I'll revert to self hosted files.

**Comment by ShmoogleOsukami** [comment]
Ooof! If you want me to squash my commits let me know, I'm a messy committer. =/

File: source/index.htm.erb (hunk 3 above)
Comment (ShmoogleOsukami) [reply source/index.htm.erb]: More specifically, I can either use the <abbr> tag or a more fancy approach, use bootstrap tooltips

**Comment by barton2526** [comment]
utACK.  @grctest ?

**Comment by ShmoogleOsukami** [comment]
Shoot.. Did I mess this PR up? I don't think those commits from 3 days ago are ment to be there =x
